diags_batch.csh

```diff
@@ -27,6 +27,8 @@
 # It's helpful to use the $casename here.
 #PBS  -o diags_batch.eo
 #PBS  -j oe 
+# Run this after st_archive finishes with no errors.
+#PBS  -W depend=afterok:2290681.chadmin1.ib0.cheyenne.ucar.edu
```

Comment:
pretty sneaky

Reply:
This did create a merge conflict ... your (local) diags_batch.csh does not have the -k eod 
directive and it should. Adding the dependency happened to conflict with where the -k eod
was added.

Reply:
BTW ... I fixed the merge conflict on the web.